### PR TITLE
Update props passed to shipping simulator

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?
Update the props passed to the `ShippingSimulator` component.

#### What problem is this solving?
The component api has been changed, so this component is being updated to follow along.

#### How should this be manually tested?
[Access the workspace](https://lucas--storecomponents.myvtex.com/aoc-tv/p)

#### Screenshots or example usage
N/A

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by andreldsa — APPROVED** [review]

**Review by brunojdo — CHANGES_REQUESTED** [review]
Make `README.md` great again. :trump:

**Review by brunojdo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# product-details
-VTEX Product Details app
+# Product Details
+
+Product Details is a canonical component that any VTEX app can import.
+
+To import it into your code:
+
+```js
+import ProductDetails from 'vtex.product-details/ProductDetails'
```
Comment (brunojdo) [inline README.md]: This is not a component that we import. You need to explain how to use as an `ExtensionPoint`. Could you?

**Review by brunojdo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lucasecdb) [reply README.md]: sure!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+Product Details is a canonical component that any VTEX store can use.
+
+## Usage
+
+To use it, you need to add the dependency in your `manifest.json`
+
+```json
+{
+  "dependencies": {
+    "vtex.product-details": "0.2.x"
```
Comment (brunojdo) [inline README.md]: 0.x

**Review by brunojdo — APPROVED** [review]

**Pull request merged**